crates/ruff_workspace/src/configuration.rs

```diff
@@ -1077,18 +1082,19 @@ impl FormatConfiguration {
 
     #[must_use]
     #[allow(clippy::needless_pass_by_value)]
-    pub fn combine(self, other: Self) -> Self {
+    pub fn combine(self, config: Self) -> Self {
```

Comment:
(This is called `config` in the `LinterConfiguration` impl, so made it consistent.)